estate/models/estate_property_offer.py

```diff
@@ -0,0 +1,42 @@
+from odoo import api, fields, models
+
+class PropertyOffer(models.Model):
+    _name = "estate.property.offer"
+    _description = "Property Offer"
+
+    price = fields.Float()
+    status = fields.Selection([('accepted', 'Accepted'), ('refused', 'Refused')], copy=False)
+    partner_id = fields.Many2one("res.partner", string='Partner', required=True)
+    property_id = fields.Many2one("estate.property", string='Property', required=True)
+    validity = fields.Integer('Validity', default=7)
+    date_deadline = fields.Date('Validity Date', compute='_compute_validity_date', inverse='_compute_validity_days')
+
+    @api.depends("validity")
+    def _compute_validity_date(self):
+        for record in self:
+            if record.create_date:
+                record.date_deadline = fields.Date.add(record.create_date, days=record.validity)
+            else:
+                record.date_deadline = fields.Date.add(fields.Date.today(), days=record.validity)
+
+    def _compute_validity_days(self):
+        for record in self:
+            if record.create_date:
+                record.validity = (record.date_deadline - record.create_date.date()).days
```

Comment:
```suggestion
                record.validity = (record.date_deadline - record.create_date).days
```
Not sure if you need it here. If you tried like this before and it didn't work then it's okay.

Reply:
I had an error without it because record.date_deadline was a datetime.date and record.create_date was a datetime.datetime.